<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Otter::InputPreferencesPage</class>
 <widget class="QWidget" name="Otter::InputPreferencesPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="keyboardTab">
      <attribute name="title">
       <string>Keyboard</string>
      </attribute>
      <layout class="QVBoxLayout" name="keyboardLayout" stretch="0,0,0,0,1">
       <item>
        <widget class="Otter::HeaderWidget" name="keyboardOptionsHeaderWidget">
         <property name="text">
          <string>Options</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="keyboardEnableSingleKeyShortcutsCheckBox">
         <property name="text">
          <string>Enable single key shortcuts</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="keyboardProfilesWidget" native="true">
         <layout class="QVBoxLayout" name="keyboardProfilesWidgetLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="Otter::HeaderWidget" name="keyboardProfilesHeaderWidget">
            <property name="text">
             <string>Profiles</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="keyboardProfilesLayout" stretch="1,0">
            <item>
             <widget class="Otter::ItemViewWidget" name="keyboardProfilesViewWidget">
              <property name="dragEnabled">
               <bool>true</bool>
              </property>
              <property name="dragDropOverwriteMode">
               <bool>false</bool>
              </property>
              <property name="dragDropMode">
               <enum>QAbstractItemView::InternalMove</enum>
              </property>
              <property name="defaultDropAction">
               <enum>Qt::MoveAction</enum>
              </property>
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::SingleSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
              <attribute name="headerVisible">
               <bool>false</bool>
              </attribute>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="keyboardProfilesButtonsLayout" stretch="0,0,0,0,1,0">
              <item>
               <widget class="QPushButton" name="keyboardAddProfileButton">
                <property name="text">
                 <string>Add</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="keyboardEditProfileButton">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Information…</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="keyboardCloneProfileButton">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Clone</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="keyboardRemoveProfileButton">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Remove</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="keyboardSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
               </spacer>
              </item>
              <item>
               <layout class="QHBoxLayout" name="keyboardProfilesMoveButtonsLayout" stretch="0,1,0">
                <item>
                 <widget class="QToolButton" name="keyboardMoveProfileUpButton">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="toolTip">
                   <string>Move Up</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="keyboardProfilesMoveButtonsSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Minimum</enum>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QToolButton" name="keyboardMoveProfileDownButton">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="toolTip">
                   <string>Move Down</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="Otter::HeaderWidget" name="keyboardShortcutsHeaderWidget">
         <property name="text">
          <string>Shortcuts</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="keyboardShortcutsLayout">
         <item>
          <layout class="QVBoxLayout" name="keyboardShortcutsViewLayout">
           <item>
            <widget class="Otter::LineEditWidget" name="keyboardShortcutsFilterLineEditWidget">
             <property name="placeholderText">
              <string>Search…</string>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Otter::ItemViewWidget" name="keyboardShortcutsViewWidget">
             <property name="contextMenuPolicy">
              <enum>Qt::NoContextMenu</enum>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::CurrentChanged|QAbstractItemView::SelectedClicked</set>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <property name="iconSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <attribute name="headerDefaultSectionSize">
              <number>150</number>
             </attribute>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QWidget" name="keyboardShortcutsButtonsWidget" native="true">
           <layout class="QVBoxLayout" name="keyboardShortcutsButtonsLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="keyboardAddShortcutButton">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="keyboardDisableShortcutButton">
              <property name="text">
               <string>Disable</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="keyboardShortcutParametersButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Parameters…</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="keyboardRemoveShortcutButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Remove</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="keyboardShortcutsSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::MinimumExpanding</enum>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mouseTab">
      <attribute name="title">
       <string>Mouse</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Otter::ItemViewWidget</class>
   <extends>QTreeView</extends>
   <header>src/ui/ItemViewWidget.h</header>
  </customwidget>
  <customwidget>
   <class>Otter::LineEditWidget</class>
   <extends>QLineEdit</extends>
   <header>src/ui/LineEditWidget.h</header>
  </customwidget>
  <customwidget>
   <class>Otter::HeaderWidget</class>
   <extends>QLabel</extends>
   <header>src/ui/HeaderWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
